website/src/pages/accelerate-dev-onboarding.tsx

```diff
@@ -4,16 +4,16 @@ import { Link, PageProps } from 'gatsby'
 import Layout from '../components/Layout'
 import { FormLegal } from '../components/FormLegal'
 import { useHubSpot } from '../hooks/hubSpot'
+import { useChiliPiper } from '../hooks/chiliPiper'
+import { buttonStyle, buttonLocation } from '../tracking'
 
 const AccelerateDevOnboarding: FunctionComponent<PageProps> = props => {
-    ;['topForm', 'bottomForm'].forEach(id => {
-        useHubSpot({
-            portalId: '2762526',
-            formId: '98187d3b-d8a9-43e2-bb95-d93dd029c688',
-            targetId: id,
-            chiliPiper: true,
-        })
+    useHubSpot({
+        portalId: '2762526',
+        formId: '98187d3b-d8a9-43e2-bb95-d93dd029c688',
+        targetId: 'topForm',
     })
+    useChiliPiper()
```

Comment:
I tested this page and a few others and the ChiliPiper scheduler doesn't run on submit

Reply:
@bretthayes should be good now! Tested on a couple of pages and the scheduler opens up on form submit.